docs/tutorials/google/echoes.ipynb

```diff
@@ -108,22 +108,12 @@
    "metadata": {
```

Comment:
<p>maybe we could seed the global numpy random generator to reduce churn in the output cells like this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
(reduce future churn, that is)